src/muxer/muxer_pass.c

```diff
@@ -446,10 +447,34 @@ pass_muxer_reconfigure(muxer_t* m, const struct streaming_start *ss)
     pm->pm_dst_onid  = pm->pm_src_onid;
   }
   pm->pm_pmt_pid     = ss->ss_pmt_pid;
+  pm->pm_rewrite_pmt = !!pm->m_config.u.pass.m_rewrite_pmt;
   pm->pm_rewrite_sdt = !!pm->m_config.u.pass.m_rewrite_sdt;
   pm->pm_rewrite_nit = !!pm->m_config.u.pass.m_rewrite_nit;
   pm->pm_rewrite_eit = !!pm->m_config.u.pass.m_rewrite_eit;
 
+  if (pm->pm_pmt_pid == DVB_SDT_PID && pm->pm_rewrite_pmt) {
+    tvhwarn(LS_PASS, "PMT PID shared with A/V, rewrite disabled");
+    pm->pm_rewrite_pmt = 0;
+  }
+  if (pm->pm_pmt_pid == DVB_SDT_PID && pm->pm_rewrite_sdt) {
+    tvhwarn(LS_PASS, "PMT PID shared with A/V, rewrite disabled");
```

Comment:
Duplicate block with previous. Just correct condition to `pm->pm_pmt_pid == DVB_SDT_PID && pm->pm_rewrite_sdt`.